docs/dxp/7.2.x/en/platform/app-builder/creating-applications-with-app-builder.md

```diff
@@ -0,0 +1,69 @@
+# Creating Applications with App Builder
+
+Objects are the foundation of your App Builder applications. They define all of the data fields that can be contained in the application. To create applications based on an object, you have four tasks: create a custom object, create at least one form view, create at least one table view, and deploy the application.
+
+- [Creating a custom object](#creating-a-custom-object)
+
+- [Creating form views](#creating-form-views)
+
+- [Creating table views](#creating-table-views)
+
+- [Deploying the application](#deploying-the-application)
+
+## Creating a Custom Object
+
+To create custom objects, 
+
+1.  Navigate to Control Panel &rarr; App Builder &rarr; Custom Objects, and click the add button.
+
+1.  Enter a name for the object.
+
+1.  To create a form view for the object and define the data fields present in the object, leave the _Continue and Create Form View_ setting checked and click _Continue_.
+
+    ![Creating a custom object is the first step to writing an App Builder application.](./images/app-builder-custom-object.png)
+
+Define the data present in your custom object by creating a form view.
```

Comment:
Again, I feel like saying "data" we may make users think that this is referring to the actual entries/records. Maybe replace this with "properties", "structure" or "Data definition"?

Reply:
@matheus-xavier I agree, but I propose to use _data fields_ instead. What do you think?

Reply:
_data fields_ is great! let's go with it.